slave/slave_channels.go

```diff
@@ -0,0 +1,43 @@
+package slave
+
+import (
+	"fmt"
+
+	"github.com/aaronang/cong-the-ripper/lib"
+)
+
+func addTask(task lib.Task, s *Slave) {
```

Comment:
I moved the methods on purpose out of `slave.go`. Otherwise the slave will be crowded with methods, and I think this is a bit cleaner.